DigitalLearningSolutions.Web.Tests/Helpers/DurationFormattingHelperTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.Helpers
+{
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.Tests.TestHelpers;
+    using DigitalLearningSolutions.Web.ViewModels.LearningMenu;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    class DurationFormattingHelperTests
+    {
+        [Test]
+        public void FormatDuration_should_have_averageDuration_for_0_minutes()

```

Comment:
Very minor but this formatting doesn't really have anything to do with average as it could be used for any duration, and we possibly do use it for the time spent as well? Either way I think it would be better to avoid talking about average duration here as it could be confusing 🙂

Reply:
Yes I agree, that was left over from the refactoring and I didn't think rename them. Fixed in 4476c83 🙂